.changeset/big-cherries-destroy.md

```diff
@@ -0,0 +1,5 @@
+---
+'@noaignite/react-centra-checkout': patch
+---
+
+fix(react-centra-checkout): add disclaimer regarding `Context.initialSelection`
```

Comment:
thought: Although nothing wrong, the changeset is not meant to be conventional commits :)